src/components/mgt-people/mgt-people.ts

```diff
@@ -288,7 +315,16 @@ export class MgtPeople extends MgtTemplatedComponent {
 
       if (provider && provider.state === ProviderState.SignedIn) {
         const graph = provider.graph.forComponent(this);
+        const betaGraph = BetaGraph.fromGraph(graph);
+
+        // populate presence for people only if userIds array exist
+        if (this.showPresence && this.userIds) {
```

Comment:
You should be able to move this after the people are fetched and then get the presence for people - should work for any people, not just for people fetched by user-id